src/plugins/kernel_install/utils.sh

```diff
@@ -45,10 +45,12 @@ function cmd_manager()
 
 function command_exists()
 {
-  local command="$1"
-  local package=${command%% *}
+  local cmd="$1"
+  local package=${cmd%% *}
 
   if [[ ! -x "$(command -v "$package")" ]]; then
+    # Fallback
+    [[ -x "/usr/sbin/${package}" ]] && return 0
     return 22 # EINVAL
   fi
   return 0
```

Comment:
This copy-and-paste solution is a terrible solution. We need to copy it because in the remote deploy kw may not be installed in the target machine. In the future, we need to find a way to send that code without duplicating it.